config/application-OS-dev.properties

```diff
@@ -7,6 +7,13 @@ repeat.maxRuns=1
 # set to read the input grades from a file not the ESB.
 #io.getGradeIO=./config/getGrades-TEST-A.json
 # set to write the output to a file rather than the ESB.
-io.putGradeIO=/opt/spe_persist/putGrades-OS-dev.txt
+#io.putGradeIO=/opt/spe_persist/putGrades-OS-dev.txt.XXX
+
+email.alwaysMailReport=TRUE
+logging.level.javax.mail=DEBUG
+logging.level.com.sun.mail=DEBUG
+
+# for testing token renewal.
+io.tokenRenewalFrequency=5
```

Comment:
Why 5? What's min and max for this number?

Reply:
The value in production should be 0.  The lower limit is 0 (to inactivate it).  There is no max value.  The value 5 is used here to ensure the code will try token renewal even when processing small amounts of data.  If we don't provide something internal to provoke token renewal there isn't any reasonable way to test the code short of setting up a mock web service.